examples/stable/elasticsearch/blueprint-v2/elasticsearch-blueprint.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: elasticsearch-blueprint
+actions:
+  backup:
+    type: StatefulSet
```

Comment:
should we remove these `type` fields now. I don't think we are using them anywhere.

Reply:
We will address this in separate PR as we may want to remove it from older BPs as well